src/util/GestureHandler.as

```diff
@@ -133,10 +133,12 @@ public class GestureHandler {
 			}
 		}
 		if (carriedObj && scrollTarget && (getTimer() - scrollStartTime) > SCROLL_MSECS && (scrollXVelocity || scrollYVelocity)) {
-			scrollTarget.contents.x = Math.min(0, Math.max(-scrollTarget.maxScrollH(), scrollTarget.contents.x + scrollXVelocity));
-			scrollTarget.contents.y = Math.min(0, Math.max(-scrollTarget.maxScrollV(), scrollTarget.contents.y + scrollYVelocity));
-			scrollTarget.constrainScroll();
-			scrollTarget.updateScrollbars();
+			if (!scrollTarget is ScriptsPane) {
```

Comment:
Yep -- looks like you're right. Thanks for bringing this to my attention!
I'll see if I can find a quick fix. If not I'll revert this change for now.